### ISSUE TITLE
Fails with nightly 2018-11-11: llvm-nm: No such file or directory

### ISSUE THREAD

**Issue by andersk** [mozilla/nixpkgs-mozilla#134]
```console
$ nix-shell --pure -p '(pkgs.rustChannelOf { date = "2018-11-10"; channel = "nightly"; }).rust'

[nix-shell:~]$ rustc --version
rustc 1.32.0-nightly (36a50c29f 2018-11-09)

[nix-shell:~]$ exit

$ nix-shell --pure -p '(pkgs.rustChannelOf { date = "2018-11-11"; channel = "nightly"; }).rust'
...
install: installing component 'llvm-tools-preview'
install: copying file /nix/store/jyxf3732rzd7h3dx8sriy8i0myrp8ji1-rust/lib/rustlib/x86_64-unknown-linux-gnu/bin/llvm-nm
cp: -r not specified; omitting directory '/build/rust-nightly-x86_64-unknown-linux-gnu/llvm-tools-preview/lib/rustlib/x86_64-unknown-linux-gnu/bin/llvm-nm'
chmod: cannot access '/nix/store/jyxf3732rzd7h3dx8sriy8i0myrp8ji1-rust/lib/rustlib/x86_64-unknown-linux-gnu/bin/llvm-nm': No such file or directory
install: error: file creation failed. see logs at '/nix/store/jyxf3732rzd7h3dx8sriy8i0myrp8ji1-rust/lib/rustlib/install.log'
builder for '/nix/store/ij2psjjbr2wbxsvm46xmhprjv9j9h7rc-rust.drv' failed with exit code 1
cannot build derivation '/nix/store/vp43w70k6ib43w8qajqqs7s8aspsikp9-rust-1.32.0-nightly-2018-11-10-6e9b84296.drv': 1 dependencies couldn't be built
error: build of '/nix/store/vp43w70k6ib43w8qajqqs7s8aspsikp9-rust-1.32.0-nightly-2018-11-10-6e9b84296.drv' failed
```

**Comment by Softsapiens** [comment]
+1

**Comment by bkchr** [comment]
That is an upstream problem: https://github.com/rust-lang/rust/issues/55866

**Comment by bkchr** [comment]
As long as upstream is not fixed, I use the following fix:
```
diff --git a/rust-overlay.nix b/rust-overlay.nix
index e1f771c..3d26354 100644
--- a/rust-overlay.nix
+++ b/rust-overlay.nix
@@ -131,7 +131,7 @@ let
           # This code is inspired by patchelf/setup-hook.sh to iterate over all binaries.
           installPhase = ''
             patchShebangs install.sh
-            CFG_DISABLE_LDCONFIG=1 ./install.sh --prefix=$out --verbose
+            CFG_DISABLE_LDCONFIG=1 ./install.sh --prefix=$out --verbose --without=llvm-tools-preview
 
             setInterpreter() {
               local dir="$1"
```

**Comment by andersk** [comment]
This was evidently fixed by commit a26d388082fcd470fa05e3276c9405dc306fdf16. For the new problem with 2019-01-13 and later, see #148.
